pkg/controller/controller.go

```diff
@@ -125,6 +125,7 @@ func registerEventHandlers(c *config.Config, notifiers []notify.Notifier, resour
 func sendEvent(obj, oldObj interface{}, c *config.Config, notifiers []notify.Notifier, kind string, eventType config.EventType) {
 	// Filter namespaces
 	objectMeta := utils.GetObjectMetaData(obj)
```

Comment:
We can get object name and namespace with `unstructured.GetName()` and `unstructured.GetNamespace()`